cllc-public-app/ClientApp/src/app/components/capacity-table/capacity-table.component.ts

```diff
@@ -99,6 +102,7 @@ export class CapacityTableComponent extends BaseControlValueAccessor<ServiceArea
     validate({ value }: FormControl) {
         let isValid = true;
         this.areasArr.controls.forEach((row, index) => {
+            console.log(row);
```

Comment:
Can you remove this extra logging statement please

Reply:
fixed. sorry about that i screwed up a merge that reverted some things. good catch